includes/admin/views/html-notice-require-compat-plugin.php

```diff
@@ -22,7 +22,7 @@
 	<p>
 		<?php
 		/*Translators: Warning to delete WooCommerce.*/
-		echo sprintf( wp_kses( __( 'Make sure to <a href="%s">delete WooCommerce</a> first before installing and activating the Compatibility Plugin.', 'classic-commerce' ), array( 'a' => array( 'href' => array() ) ) ), esc_url( admin_url( 'plugins.php' ) ) );
+		echo sprintf( wp_kses( __( 'Make sure to <a href="%s">delete WooCommerce</a> first before installing and activating the Compatibility Plugin. Deleting WooCommerce will not remove your products, settings and other data.', 'classic-commerce' ), array( 'a' => array( 'href' => array() ) ) ), esc_url( admin_url( 'plugins.php' ) ) );
```

Comment:
This change should be backed out and done in a separate PR since it's not related to the purpose of this PR.

Reply:
Yes. Understood.

Reply:
This change is mentioned in the PR submission but agree it probably needs its own PR.